packages/meditrak-server/src/database/models/Option.js

```diff
@@ -4,14 +4,21 @@
  **/
 
 import { DatabaseModel, DatabaseType, TYPES } from '@tupaia/database';
-import { hasContent } from '@tupaia/utils';
 
 class OptionType extends DatabaseType {
   static databaseType = TYPES.OPTION;
 
   static fieldValidators = new Map()
     .set('value', [
-      hasContent,
+       async (value) => {
+        const checkIsEmpty = value => value === undefined || value === null || value.length === 0;
+          if (checkIsEmpty(value)) {
+            return 'Value cannot be empty'; 
+          }
+          else {           
+           return null;
+          };
+      },
```

Comment:
```suggestion
       value => {
        try {
          return hasContent(value) && null;
        } catch (error) {
          return error.message;
        }
      },
```

Reply:
We could refactor the fix to use a try/catch block which would have the advantage of re-using the `hasContent` method from utils but on the other hand breaks the pattern of the other validators.